metrics/local.go

```diff
@@ -208,9 +208,11 @@ func (b *LocalBackend) Stop() {
 // "name|tag1=value1|...|tagN=valueN", where tag names are
 // sorted alphabetically.
 func getKey(name string, tags map[string]string) string {
-	var keys []string
+	keys := make([]string, len(tags))
+	var i int
 	for k := range tags {
-		keys = append(keys, k)
+		keys[i] = k
+		i++
```

Comment:
that case was different, there you already had an index variable incremented as part of the loop. So it made the code cleaner and faster. Here I don't think it's faster, just uglier due to `i` being in the outside scope.

Anyway, fine either way, this isn't a performance critical piece.